docs/topics/ref_known-issues-7-2-0.adoc

```diff
@@ -0,0 +1,43 @@
+:_template-generated: 2024-12-04
+:_mod-docs-content-type: REFERENCE
+
+[id="known-issues-7-2-0_{context}"]
+= Known issues
+
+This section provides highlighted known issues in {ProductFullName} version 7.2.0.
+
+.{ProductShortName} [CLI] analysis using the Container CLI fails on Windows
+
+{ProductShortName} [CLI] analysis using the Container CLI fails on Microsoft Windows and returns a `provider does not have dependency capability` error. This failure is caused by the existence of the `.metadata` folder located inside the folder where the analysis is run. Therefore, the workaround is to delete this folder. This issue is planned to be resolved in {ProductShortName} 7.2.1. link:https://issues.redhat.com/browse/MTA-4410[(MTA-4410)]
+
+.{ProductShortName} [CLI] analysis using the Containerless CLI fails on Windows
+
+{ProductShortName} [CLI] analysis using the Containerless CLI fails on Microsoft Windows and returns a `provider does not have dependency capability` error. This failure is caused by the existence of the `.metadata` folder located inside the folder where the analysis is run. Therefore, the workaround is to delete this folder. This issue is planned to be resolved in {ProductShortName} 7.2.1. link:https://issues.redhat.com/browse/MTA-4416[(MTA-4416)]
+
+.{ProductShortName} CLI analysis is extremely slow on Windows
+
+{ProductShortName} CLI analysis does not function as expected on Microsoft Windows, with analysis using container runtimes being extremely slow on Windows. This issue is planned to be resolved in {ProductShortName} 7.2.1. link:https://issues.redhat.com/browse/MTA-3667[(MTA-3667)]
+
+.Analysis of the daytrader application does not show dependencies
+
+The analysis of the daytrader application does not show dependencies. This failure to display dependencies is caused by the {ProductShortName} Operator using Maven 3.6, which sometimes fails to fetch dependencies. This issue is planned to be resolved in {ProductShortName} 7.2.1. link:https://issues.redhat.com/browse/MTA-4169[(MTA-4169)]
+
+.{ProductShortName} CLI bulk analysis does not function on Windows
+
+{ProductShortName} CLI bulk analysis, performed by using the `--bulk` option, does not function on Microsoft Windows. This issue is planned to be resolved in {ProductShortName} 7.2.1. link:https://issues.redhat.com/browse/MTA-4307[(MTA-4307)]
+
+.{ProductShortName} Containerless CLI does not fire XML custom rules
```

Comment:
@ibragins this bug has been verified as well right? I think this section can be removed...